index.html

```diff
@@ -464,6 +465,51 @@ <h2>
       Integration with Credential Management API
     </h2>
     <aside class="issue" data-number="65"></aside>
+    <h3>
+      The `<dfn>clientAllowsProtocol()</dfn>` method
+    </h3>
+    <p>
+      The {{DigitalCredential/clientAllowsProtocol()}} method allows digital
+      credential verifiers and issuers to determine which 
+      [=digital credential/exchange protocols=] the client supports for
+      interactions with [=credential providers=].
+    </p>
+    <p class="note">
+      This method does not convey [=digital credential/request protocol=] or 
+      [=digital credential/issuance protocol=] support in the [=client device=]
+      or [=credential provider=].
+    </p>
+    <p>
+        User agents SHOULD NOT vary the response value based on any information
+        about availability of hardware, presence or configuration of software,
+        wallets, credential providers, or digital credentials, or user
+        configuration or preferences. If the response value varied, the user
+        agent would introduce risks both of fingerprinting and of silently
+        revealing other details about user behavior or configuration. The
+        response value SHOULD vary only by user agent major version and
+        indicate whether the browser supports distributing requests with that
+        protocol to underlying platform or provider.
```

Comment:
Should we stick to referring to client here for example?
```suggestion
        revealing other details about user behavior or configuration. The
        response value SHOULD vary only by client's major version and
        indicate whether the client supports distributing requests with that
        protocol to underlying platform or provider on the client device.
```

Reply:
For sake of progress, a definition of "client" has been proposed in #271 for us to talk about at the next WG meeting.